//HintName: Test.Stuff.CatchException.g.cs
// <auto-generated/>
#nullable enable
namespace Test;
internal partial class Stuff
{
    public static void CatchException()
    {
        try
        {
        }
        catch (global::System.OperationCanceledException)
        {
        }
    }
}
